### PR TITLE
[tests-only] [full-ci] Add test for default depth activity filter

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
## Description
Added test for default depth activity filter i.e. `depth:-1`

## Related Issue
- Part of https://github.com/owncloud/ocis/issues/9543
- https://github.com/owncloud/ocis/issues/9850

## How Has This Been Tested?

- locally
- CI :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiActivities/activities.feature [hunk 1]
```diff
@@ -1327,6 +1327,135 @@ Feature: check activities
...
+                        "variables": {
+                          "type": "object",
+                          "required": ["resource", "space", "user"],
+                          "properties": {
+                            "resource": {
+                              "type": "object",
+                              "required": ["id", "name"],
+                              "properties": {
+                                "name": {
+                                  "const": "textfile.txt"
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiActivities/activities.feature:1438]:
```suggestion
                              "required": ["id", "name"],
                              "properties": {
                                "name": {
                                  "const": "textfile.txt"
```
did I missed id or its really not present in json scheme?

Comment (Salipa-Gurung) [reply tests/acceptance/features/apiActivities/activities.feature:1438]: There's an open issue related to missing resource id so, we are not check for id pattern.

File: tests/acceptance/features/apiActivities/activities.feature [hunk 2]
```diff
@@ -1327,6 +1327,135 @@ Feature: check activities
       }
       """
 
+  @issue-9850
+  Scenario: check activity with default depth filter
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:1331]:
```suggestion
  Scenario: check activity with -1 depth filter
```

**Review by saw-jan — APPROVED** [review]

File: tests/acceptance/features/apiActivities/activities.feature (hunk 1 above)
Comment (saw-jan) [reply tests/acceptance/features/apiActivities/activities.feature:1438]: whether the id thing will be fixed or not is unknown

Comment (saw-jan) [reply tests/acceptance/features/apiActivities/activities.feature:1438]: but it is possible for us to check the empty id

Comment (saw-jan) [reply tests/acceptance/features/apiActivities/activities.feature:1438]: @Salipa-Gurung Please, check empty id in next PR. We will know whenever there is change in id in the future

Comment (Salipa-Gurung) [reply tests/acceptance/features/apiActivities/activities.feature:1438]: > but it is possible for us to check the empty id

yes, we can

**Pull request merged**